### PR TITLE
Release 4.3.0

### PR DESCRIPTION
**Pull request by jonmmease**
Release branch for 4.3.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+ - Use the default template's background color for displaying color swatches ([#1872](https://github.com/plotly/plotly.py/pull/1872)). Special thanks to [@joelostblom](https://github.com/joelostblom) for this contribution!
+ 
+### Added
+ - Added image trace type ([plotly.js#4289](https://github.com/plotly/plotly.js/pull/4289), [plotly.js#4307](https://github.com/plotly/plotly.js/pull/4307), [plotly.js#4313](https://github.com/plotly/plotly.js/pull/4313), [plotly.js#4319](https://github.com/plotly/plotly.js/pull/4319))
+ - Added matplotlib-style `plotly.express.imshow` convenience function to display images and heatmaps ([#1855](https://github.com/plotly/plotly.py/pull/1855), [#1885](https://github.com/plotly/plotly.py/pull/1885))
+ - Added matplotlib-style `simple_white` template ([#1864](https://github.com/plotly/plotly.py/pull/1864))
+ - Added support for using an externally managed orca server for image export features ([#1850](https://github.com/plotly/plotly.py/pull/1850)). Special thanks to [@miriad](https://github.com/miriad) for this contribution.
```
Comment (emmanuelle) [inline CHANGELOG.md]: the `facet_wrap` argument of `px` should be part of the changelog if I'm not mistaken

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
...
+ - Update the `add_annotations`/`add_shapes`/`add_images` methods to no longer default to adding objects in paper coordinates. This allows plotly.js to determine the default reference frame based on context ([#1888](https://github.com/plotly/plotly.py/pull/1888))
+ - Use the default template's background color for displaying color swatches ([#1872](https://github.com/plotly/plotly.py/pull/1872)). Special thanks to [@joelostblom](https://github.com/joelostblom) for this contribution!
+ 
+### Added
+ - Added image trace type ([plotly.js#4289](https://github.com/plotly/plotly.js/pull/4289), [plotly.js#4307](https://github.com/plotly/plotly.js/pull/4307), [plotly.js#4313](https://github.com/plotly/plotly.js/pull/4313), [plotly.js#4319](https://github.com/plotly/plotly.js/pull/4319))
+ - Added matplotlib-style `plotly.express.imshow` convenience function to display images and heatmaps ([#1855](https://github.com/plotly/plotly.py/pull/1855), [#1885](https://github.com/plotly/plotly.py/pull/1885))
+ - Added matplotlib-style `simple_white` template ([#1864](https://github.com/plotly/plotly.py/pull/1864))
```
Comment (nicolaskruchten) [inline CHANGELOG.md]: let's thank @joelostblom for this one also

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,22 @@
...
+ - Added support for using an externally managed orca server for image export features ([#1850](https://github.com/plotly/plotly.py/pull/1850)). Special thanks to [@miriad](https://github.com/miriad) for this contribution.
+
```
Comment (nicolaskruchten) [inline CHANGELOG.md]: Let's also mention "improved docstrings [#1835](https://github.com/plotly/plotly.py/pull/1835) and [#1837](https://github.com/plotly/plotly.py/pull/1837)"

File: CHANGELOG.md (hunk 1 above)
Comment (nicolaskruchten) [reply CHANGELOG.md]: This one is https://github.com/plotly/plotly.py/pull/1838

File: CHANGELOG.md (hunk 2 above)
Comment (jonmmease) [reply CHANGELOG.md]: Thanks for catching that oversight!

**Pull request merged**